### ISSUE TITLE
[Bug]:RangeCalendarCellTrigger contains incorrect aria-selected attribute 

### ISSUE THREAD

**Issue by wolandec** [radix-vue/radix-vue#2261]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Reka UI version: 2.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

https://www.reka-ui.com/docs/components/date-range-picker#date-range-picker

### Steps to reproduce

Open DateRangePicker, select any dates and run axe a11y report.

### Describe the bug

The issue is that role of the element is set to button, but `aria-selected` attribute is not allowed on buttons, it should be `aria-pressed`. In this case axe a11y test passes and what even more important Voiseover can spell selected on the button if the day is selected, also it mention the type of button `toggle-button` 

### Expected behavior

a11y test should pass and Voiseover spell the right information

### Context & Screenshots (if applicable)

<img width="1153" height="760" alt="Image" src="https://github.com/user-attachments/assets/80e11c8e-cc19-4294-bbf7-e8d98116e871" />

**Comment by wolandec** [comment]
I opened [PR](https://github.com/unovue/reka-ui/pull/2265) to fix this. @zernonia please let me know your thoughts and if we can merge it.
